src/components/Contactlist.js

```diff
@@ -0,0 +1,88 @@
+import React, {useState} from 'react'
+import '../App.css';
+import contactsJson from '../contacts.json';
+
+
+const firstFiveContacts = contactsJson.slice(0,5)
+
+function ContactList(props) {
+    
+    const [contacts, setContacts] = useState(firstFiveContacts)
+
+   
+     const randomContact = () => {
+        
+        const contacstOfFive =[...contacts]
+        const randomContactFromCopy= contactsJson[Math.floor(Math.random() * contactsJson.length+1)]
+        contacstOfFive.push(randomContactFromCopy)
+        setContacts(contacstOfFive)
+       // console.log(randomContactFromCopy);
+
+     }
+
+     const sortByName = () => {
+        const contacstOfFive =[...contacts]
+        contacstOfFive.sort((a, b) => a.name.localeCompare(b.name));
+        setContacts(contacstOfFive)
+     }
+
+     const sortByPopularity = () => {
+        const contacstOfFive =[...contacts]
+        contacstOfFive.sort(function(a,b){
+            
+            if (a.popularity > b.popularity){
+                return -1
+            } 
+            if (a.popularity < b.popularity){
+               return 1
+            }
+            return 0
+
+        });
+        setContacts(contacstOfFive)
+     }
+
+     const deleteContact = (delContactItem) => {
```

Comment:
If you process my first comment, in this function the element can be added to the contactsJson